lib/dor/workflow/client/lifecycle_routes.rb

```diff
@@ -57,15 +63,13 @@ def filter_milestone(lifecycle_doc, milestone_name)
         #     <milestone date="2010-06-15T16:08:58-0700">released</milestone>
         #   </lifecycle>
         #
-        def query_lifecycle(repo, druid, active_only: false, version: nil)
+        def query_lifecycle(repo, druid, active_only:, version: nil)
           req = "#{repo}/objects/#{druid}/lifecycle"
-          req += if version
-                   "?version=#{version}"
-                 elsif active_only
-                   '?active-only=true'
-                 else
-                   ''
-                 end
+          params = []
+          params << "version=#{version}" if version
+          params << 'active-only=true' if active_only
+          req += "?#{params.join('&')}" unless params.empty?
```

Comment:
I have a minor preference for `if params.any?` or `unless params.empty?` but nbd.

Reply:
Those methods do different things:
```
irb(main):004:0> [false].any?
=> false
irb(main):005:0> ![false].empty?
=> true
```

Reply:
argh